src/core/__tests__/ReactMultiChildReconcile-test.js

```diff
@@ -107,13 +107,14 @@ var FriendsStatusDisplay = React.createClass({
     return res;
   },
   render: function() {
-    var children = null;
+    var children = [];
     var key;
     for (key in this.props.usernameToStatus) {
       var status = this.props.usernameToStatus[key];
-      children = children || {};
-      children[key] = !status ? null :
-          <StatusDisplay ref={key} status={status} />;
+      children.push(
+        !status ? null :
```

Comment:
Not sure what this test is testing, but perhaps it makes sense to skip the null completely?